internal/cmd/deploy/deploy.go

```diff
@@ -61,6 +86,33 @@ The bucket's name will be of the format rain-artifacts-<AWS account id>-<AWS reg
 		// Parse params
 		parsedParams := ListToMap("param", params)
 
+		if len(cliJSONFile) != 0 {
+			jsonFile, err := os.Open(cliJSONFile)
+			if err != nil {
+				panic(ui.Errorf(err, "error opening cli json file '%s'", cliJSONFile))
+			}
+			defer jsonFile.Close()
+
+			byteValue, err := ioutil.ReadAll(jsonFile)
+			if err != nil {
+				panic(ui.Errorf(err, "unable to read cli json file '%s'", cliJSONFile))
+			}
+
+			var cliInput cliInputJSON
+			err = json.Unmarshal([]byte(byteValue), &cliInput)
```

Comment:
One of rain's goals is to not care what format CloudFormation templates you prefer, I think we should do the same with the configuration file and use the yaml parser (JSON is valid YAML).

Reply:
Good idea, I changed it to the YAML parser.